### PR TITLE
Add links that will open in iOS browser if clicked during auth.

### PR DESCRIPTION
**Pull request by brandonpollett**

### CODE REVIEW

File: Microsoft.HealthVault.Client.Ios/IosBrowserAuthBroker.cs [hunk 1]
```diff
@@ -26,6 +27,17 @@ internal class IosBrowserAuthBroker : NSObject, IBrowserAuthBroker, ISignInNavig
         private SignInViewController _signInViewController;
         private bool _isTaskComplete;
         private string _endUrlString;
+        private readonly HashSet<string> _browserLinks = new HashSet<string>
```
Comment (davidkwoods) [inline Microsoft.HealthVault.Client.Ios/IosBrowserAuthBroker.cs]: Most of these urls seem a little too specific.  The clcid and region, mobile=false and that vv one all seem like they shouldn't be part of the string.  I expect we'll need to use something like .StartsWith comparisons to be robust here.

**Comment by aviade** [comment]
:shipit:

File: Microsoft.HealthVault.Client.Ios/IosBrowserAuthBroker.cs (hunk 1 above)
Comment (aviade) [inline Microsoft.HealthVault.Client.Ios/IosBrowserAuthBroker.cs]: +1

File: Microsoft.HealthVault.Client.Ios/IosBrowserAuthBroker.cs [hunk 2]
```diff
@@ -152,5 +167,26 @@ public void ContentProcessDidTerminate(WKWebView webView)
         {
             SetTaskResult(null, new HealthServiceException(ClientResources.LoginError));
         }
+
+        [Export("webView:decidePolicyForNavigationAction:decisionHandler:")]
+        public void DecidePolicy(WKWebView webView, WKNavigationAction navigationAction, Action<WKNavigationActionPolicy> decisionHandler)
+        {
+            NSUrlComponents compoents = new NSUrlComponents(navigationAction.Request.Url, false)
```
Comment (davidkwoods) [inline Microsoft.HealthVault.Client.Ios/IosBrowserAuthBroker.cs]: >compoents [](start = 28, length = 9)

nit: typo on "compoents"

**Pull request merged**